iscan/models.py

```diff
@@ -1087,9 +1087,13 @@ def run_enrichment(self):
 class BackgroundTask(models.Model):
     task_id = models.UUIDField(primary_key=True)
     name = models.CharField(max_length=100)
-    corpus = models.ForeignKey(Corpus, on_delete=models.CASCADE)
+
+    #Allow optional corpus for script tasks
+    corpus = models.ForeignKey(Corpus, null=True, blank=True, on_delete=models.CASCADE)
```

Comment:
I'm assuming this is optional because SPADE scripts don't create corpora objects as part of their processing.  I think the better fix would be to make sure that Corpora objects are created through SPADE scripts so that they're queryable from ISCAN, see https://github.com/MontrealCorpusTools/SPADE/issues/53